builtin/game/item_entity.lua

```diff
@@ -75,7 +76,8 @@ core.register_entity(":__builtin:item", {
 			itemstring = self.itemstring,
 			always_collect = self.always_collect,
 			age = self.age,
-			dropped_by = self.dropped_by
+			dropped_by = self.dropped_by,
+			mergeable = self.mergeable
```

Comment:
Perhaps invert this and store it as `nil` if it's set to merge.  This would save more than a few bytes in almost all cases.